fineract-provider/src/main/java/org/apache/fineract/organisation/office/api/OfficesApiResource.java

```diff
@@ -176,6 +176,25 @@ public String updateOffice(@PathParam("officeId") @Parameter(description = "offi
         return toApiJsonSerializer.serialize(result);
     }
 
+    @PUT
```

Comment:
Can't we use the new serializers to do this API? i.e. return the object directly from the method, etc so that you don't need this many swagger annotations.